website/markdown/docs/dependencies/dependencies.mdx

```diff
@@ -0,0 +1,76 @@
+---
+name: Dependencies.swift (Chimera Project)
+exceprt: Learn how to define standardise the contract between the dependency managers and Tuist.
+---
+
+# Dependencies.swift (Chimera Project)
+
+Learn how to define standardise the contract between the dependency managers (Cocoapods, Carthage and SPM) and Tuist.
+
+## How to get started
+
+### Step 1: `Dependencies.swift`
+
+1. Create `Dependencies.swift` in your project in `Tuist/` directory.
+2. Run `tuist edit` to start editing your manifest files.
+3. Add `import ProjectDescription` on the top of file.
+4. Define your dependencies.
+
+Example of project structure with added `Depedencies.swift` manifest file:
+
+```bash
+AwesomeProject
+	|- App
+	|- AppTests
+	|- Tuist
+		|- Dependencies.swift
+	|- Project.swift
+```
+
+The snippet below shows an example `Dependencies.swift` manifest file:
+
+```swift
+import ProjectDescription
+
+let dependencies = Dependencies([
+    .carthage(origin: .github(path: "Alamofire/Alamofire"), requirement: .exact("5.0.4"), platforms: [.macOS])
+])
+```
+
+### Step 2: `tuist dependencies` commands
+
+1. Pull depedencies using `tuist depednecies fetch` command.
+2. You can read more about commands [here](/docs/commands/dependencies/).
+3. Tuist will save depedencies in `/Tuist/Dependencies` directory.
+
+The folder structure below shows how Tuist organize dependenies:
+
+```bash
+Tuist
+    |- Dependencies.swift # Manifest
+    |- Dependencies
+        |- graph.json # comming soon
+        |- Lockfiles
+            |- Carthage.resolved
+            |- Podfile.lock # comming soon
+            |- Package.resolved # comming soon
+        |- Carthage # stores content of `Carthage/Build` generated by `carthage`
+            |- iOS
+                |- Alamofire.framework
+            |- tvOS
+                |- Alamofire.framework
+        |- Cocoapods # comming soon
+            |- RxSwift
+        |- SwftPackageManager # comming soon
+            |- Moya
+```
+
+### Step 3: Link dependencies
+
+You can link third-party dependencies like local ones.
+
+```swift
+let target = Target(dependencies: [
+    .framework("Tuist/Dependencies/Carthage/iOS/Alamofire.framework"),
```

Comment:
I might have missed this but were not we supposed to introduce something like `.tuist(...)`, so user don't have to rely on the paths we create in `Tuist/Dependencies`

Reply:
I have another comment, not related to this PR - is the location of built frameworks from Carthage or in the future `Pods` directory put into `.gitignore` when running `tuist init`? If not, it might be something to consider

Reply:
ad. 1
It will be added in the future. Right now I wanted to reflect in documentation the current usage.
ad. 2
Yes I added it to `Gitignore.stencil`. @pepibumur suggest same thin a few PRs ago ;)

Reply:
Great, glad to hear that 🥳